hack/verify-golint.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+kube::golang::verify_go_version
+
+cd "${KUBE_ROOT}"
+
+array_contains () {
+    local seeking=$1; shift
+    local in=1
+    for element; do
+        if [[ "$element" == "$seeking" ]]; then
+            in=0
+            break
+        fi
+    done
+    return $in
+}
+
+export IFS=$'\n'
+all_packages=(
+	$(go list -f '{{.ImportPath}}' ./... | grep -v vendor | sed 's/github.com\/kubernetes\/kubernetes\///g' | sed 's/k8s.io\/kubernetes\///g')
```

Comment:
anything listed as "github.com/kubernetes/kubernetes" should probably be changed, I'm not sure why this would have hits.